test/app/views/courses/CoursesView.spec.coffee

```diff
@@ -30,4 +31,4 @@ describe 'CoursesView', ->
       view.$('.change-hero-btn').click()
       expect(view.openModalView).toHaveBeenCalled()
       args = view.openModalView.calls.argsFor(0)
-      expect(args[0] instanceof HeroSelectModal).toBe(true)
+      expect(args[0] instanceof HeroSelectModal).toBe(true)
```

Comment:
My phone is showing a change here that I can't see.  Probably nothing.

Reply:
I think just a `\n` change probably.